### PR TITLE
@mzikherman - Gene filter integration 

### PR DESCRIPTION
**Pull request by broskoski**
~~This isn't totally working yet, still need to wire up the toggling between artist and artwork modes.~~

### Artist mode:
screenshot 2017-05-24 18 29 48

### Artwork mode: 
screenshot 2017-05-24 18 23 28

### CODE REVIEW

**Comment by broskoski** [comment]
@mzikherman this is ready to go!

File: src/components/gene/contents.tsx [hunk 1]
```diff
@@ -0,0 +1,334 @@
...
+      for_sale: isForSale,
+    })
+    this.props.relay.setVariables({
+      for_sale: forSaleVar,
+      artworksSize: PageSize,
+      showArtists: false,
+    })
+  }
+
+  loadMoreArtworks() {
```
Comment (mzikherman) [inline src/components/gene/contents.tsx]: Nice

File: src/components/gene/contents.tsx [hunk 2]
```diff
@@ -0,0 +1,334 @@
...
+      <LoadMoreContainer>
+        <LoadMoreButton onClick={() => this.loadMoreArtists()}>Load More</LoadMoreButton>
+      </LoadMoreContainer>
+    )
+
+    const artistEl = (
+      <div>
+        {artistRows}
+        <SpinnerContainer>
+          {this.state.loading ? <Spinner /> : ""}
```
Comment (mzikherman) [inline src/components/gene/contents.tsx]: sweet

**Comment by mzikherman** [comment]
Overall the code is super readable and it's pretty clear whats going on. I'm down to merge and continue on (+ try to get in here and make some additions as well!), but if others have comments...now's the time!

**Comment by broskoski** [comment]
Alright, mergin!

**Pull request merged**